tck/src/main/java/com/github/bsideup/liiklus/records/tests/BackPressureTest.java

```diff
@@ -0,0 +1,52 @@
+package com.github.bsideup.liiklus.records.tests;
+
+import com.github.bsideup.liiklus.records.RecordStorageTestSupport;
+import com.github.bsideup.liiklus.records.RecordsStorage;
+import lombok.val;
+import org.junit.jupiter.api.Test;
+import reactor.test.StepVerifier;
+
+import java.time.Duration;
+
+public interface BackPressureTest extends RecordStorageTestSupport {
+
+    @Test
+    default void testSubscribeBackpressure() throws Exception {
```

Comment:
as it actually TCK, I'd suggest to add `@DisplayName` with strict human-readable title

Reply:
👍 I'll add the display names as a follow up, thanks! (need to merge asap)